sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -695,6 +695,24 @@ def get_datapath_deinit_duration(self):
         duration = self.xcvr_eeprom.read(consts.DP_PATH_DEINIT_DURATION)
         return float(duration) if duration is not None else 0
 
+    def get_module_pwr_up_duration(self):
+        '''
+        This function returns the duration of module power up
+        '''
+        if self.is_flat_memory():
+            return 0
+        duration = self.xcvr_eeprom.read(consts.MODULE_PWRUP_DURATION)
+        return float(duration) if duration is not None else 0
+
+    def get_module_pwr_down_duration(self):
+        '''
+        This function returns the duration of module power down
+        '''
+        if self.is_flat_memory():
+            return 0
+        duration = self.xcvr_eeprom.read(consts.MODULE_PWRDN_DURATION)
+        return float(duration) if duration is not None else 0
+
```

Comment:
@chiourung we are not using these APIs for [set_lpmode](https://github.com/sonic-net/sonic-platform-common/blob/master/sonic_platform_base/sonic_xcvr/api/public/cmis.py#L957). can you fix that?

Reply:
I will raising new PR for set_lpmode